src/main/java/gregtech/api/unification/material/MarkerMaterials.java

```diff
@@ -83,22 +84,22 @@ public static MarkerMaterial valueOf(String string) {
      * Circuitry, batteries and other technical things
      */
     public static class Tier {
-        public static final Material Primitive = new MarkerMaterial("primitive");
-        public static final Material Basic = new MarkerMaterial("basic");
-        public static final Material Good = new MarkerMaterial("good");
-        public static final Material Advanced = new MarkerMaterial("advanced");
-        public static final Material Extreme = new MarkerMaterial("extreme");
-        public static final Material Elite = new MarkerMaterial("elite");
-        public static final Material Master = new MarkerMaterial("master");
-        public static final Material Ultimate = new MarkerMaterial("ultimate");
-        public static final Material Super = new MarkerMaterial("super");
-        public static final Material Infinite = new MarkerMaterial("infinite");
-
-        public static Material Ultra = new MarkerMaterial("ultra");
-        public static Material Insane = new MarkerMaterial("insane");
-        public static Material Epic = new MarkerMaterial("epic");
-        public static Material Legendary = new MarkerMaterial("legendary");
-        public static Material Maximum = new MarkerMaterial("maximum");
+        public static final Material ULV = new MarkerMaterial(GTValues.VN[GTValues.ULV].toLowerCase());
```

Comment:
Is this supposed to use `toLowerCase()`? That results in oredict names such as `circuitUlv`, `circuitZpm`

Reply:
`toLowerCase()` is required here as materials cannot be created with uppercase letters in their unlocalized names.